mesonbuild/interpreter/type_checking.py

```diff
@@ -633,6 +633,7 @@ def _name_suffix_validator(arg: T.Optional[T.Union[str, T.List]]) -> T.Optional[
         default={},
         since='1.2.0',
     ),
+    KwargInfo('swift_interoperability_mode', str, default='c', validator=in_set_validator({'c', 'cpp'}), since='1.9.0'),
```

Comment:
I think that there's a corresponding change that needs to go into `kwargs.py`

Reply:
Like this?